.eslintrc.cjs

```diff
@@ -12,7 +12,8 @@ module.exports = {
 		"project": "./packages/**/tsconfig.json"
 	},
 	"plugins": [
-		"@typescript-eslint"
+		"@typescript-eslint",
+    "import"
```

Comment:
inconsistent indent (two tabs on line 15, four spaces on line 16). same for line 42

Reply:
ugh, i missed this because i turned off whitespace diffs on github

Reply:
nevermind that's not at all why i missed it